.github/workflows/connect-test.yml

```diff
@@ -2,7 +2,7 @@ name: "[Test] connect"
 
 # run only if there are changes in connect or related libs paths
 on:
-  push:
+  pull_request:
```

Comment:
This should fix issues when connect tests are not run when there are changes in PR but not in last commit